src/Component/Basket/BasketElement.php

```diff
@@ -126,7 +126,7 @@ public function getProductId()
 
     public function setProductId($productId): void
```

Comment:
This is the `master` branch, you are allowed to make BC breaks and require and `int` here, by using a type declaration.

Reply:
Done